code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -437,6 +437,16 @@
 						null, span_hear("You hear the rustling of clothes."), DEFAULT_MESSAGE_RANGE, list(helper, src))
 		to_chat(helper, span_notice("You shake [src] trying to pick [p_them()] up!"))
 		to_chat(src, span_notice("[helper] shakes you to get you up!"))
+	else if(helper.zone_selected == BODY_ZONE_PRECISE_MOUTH) //Boops
+		if(HAS_TRAIT(src, TRAIT_BADTOUCH) & prob(50))
```

Comment:
shouldnt this be &&? & is the bitwise AND

Reply:
Can you elaborate on the difference, cuz this already seems to work

Reply:
Basically & compares binary when && compared.raw true/false

It's extremely preferred to use && whenever not using bitflags (which prob() doesn't, it just gives a true or false), because while & can work, it doesn't always

Reply:
this absolutely doesn't work the way you think it does, niko is correct https://www.byond.com/docs/ref/#/operator/&&